streaming/multimodal/convert/laion/laion400m/convert_and_upload.py

```diff
@@ -145,7 +145,11 @@ def convert(parquet_filename: str, mds_dirname: str, hashes: List[str]) -> None:
     compression = None  # Don't compress because the vast majority of the data is JPEG.
     size_limit = None  # Put everything in one shard (1:1 mapping of parquet to MDS shards).
 
-    with MDSWriter(mds_dirname, columns, compression, hashes, size_limit) as out:
+    with MDSWriter(local=mds_dirname,
```

Comment:
nit: `split_dir` in sync with others ?

Reply:
here we have `parquet_filename` and `mds_dirname`